test/testsuite.cpp

```diff
@@ -110,7 +110,7 @@ std::string TestFixture::getLocationStr(const char * const filename, const unsig
     return ret.str();
 }
 
-std::string TestFixture::writestr(const std::string &str, bool gccStyle)
+std::string writestr(const std::string &str, bool gccStyle = false)
```

Comment:
I believe this writestr should use local linkage. Make it static.

Reply:
sorry you're right. I should have noticed this. pushed the change. There's now no change to this function in the diff.